### DISCUSSION TITLE
How can I configure LED layout for ceiling Hue lights?

### DISCUSSION THREAD

**Discussion by sofakng** [hyperion-project/hyperion.ng#1318] [General]
I've successfully configured Hyperion.ng to use Hue Entertainment, but how can I configure the positions of the lights?

For example, my Hue bulbs are located in recessed lighting cans in the ceiling.  I was hoping to use these for effects such as lightning or general ambiance.

**Comment by Lord-Grey** [comment]
See https://hyperion-project.org/forum/index.php?thread/12130-how-can-i-configure-hyperion-for-ceiling-recessed-lighting-re-led-layout/#post41382
